### PR TITLE
Enums Task Maria

### PR DESCRIPTION
**Pull request by mariastephan2186**
Enums task done

### CODE REVIEW

File: src/main/java/session13enums/examples/DirectionsTask_Maria.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    private final String description;
+
+
+    DirectionsTask_Maria(String description) {
+        this.description = description;
+
+    }
+
+    public static void main(String[] args) {
+        DirectionsTask_Maria east = DirectionsTask_Maria.EAST;
```
Comment (owynowyn) [inline src/main/java/session13enums/examples/DirectionsTask_Maria.java:18]: You could try calling this in a loop using the `DirectionsTask_Maria.values()` method and looping through the array

**Review by owynowyn — CHANGES_REQUESTED** [review]
Nice work, exactly what we were looking for but again, for an additional task, try looping through the built-in method

**Comment by mariastephan2186** [comment]
@owynowyn Made the changes to the task - added a loop through using the Values method.

**Review by owynowyn — APPROVED** [review]
Perfect! ✅

**Pull request merged**